### PR TITLE
store a copy of gallery images to avoid re-fetching

### PR DESCRIPTION
**Pull request by JosiasAurel**
#1396 

Right now, every time you perform a search the game thumbnail is re-fetched from the server. 

Now game image data is generated at build time and stored in the public folder. These image data are cached through vercel resulting in less resources transferred.

### CODE REVIEW

**Review by grymmy — APPROVED** [review]
Please test that this doesn't cause issues in worst case scenarios

File: vercel.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (grymmy) [inline vercel.json:1]: This seems very global in nature...  will it have unintended side effects?

File: src/integrations/thumbnail.ts [hunk 2]
```diff
@@ -126,28 +124,22 @@ export const get: APIRoute = async ({ url }) => {
 		}	
 	} catch (error) {
 		// If everything breaks, use a default image
```
Comment (grymmy) [inline src/integrations/thumbnail.ts:126]: tbh... I do wonder if we should instead take the approach of "if anything breaks, break"

File: src/integrations/thumbnail.ts [hunk 3]
```diff
@@ -126,28 +124,22 @@ export const get: APIRoute = async ({ url }) => {
...
-			'Access-Control-Allow-Credentials': 'true',
-			'Access-Control-Allow-Origin': '*',
-			'Access-Control-Allow-Methods': 'GET,OPTIONS',
-			'Access-Control-Allow-Headers': '*',
-			'Cache-Control': 'max-age=86400'
-		}
-	})
+	// write/overwrite image json to public folder
+	let imgFilePath = path.resolve(__dirname, `../../public/${name}.json`);
+	fs.writeFileSync(imgFilePath, JSON.stringify(thumbnail), { encoding: "utf-8" });
```
Comment (grymmy) [inline src/integrations/thumbnail.ts:137]: Please ensure this operation succeeds, and if not have it at least log something.

**Review by grymmy — APPROVED** [review]
Please resolve / respond / think about to my other comments before landing.

File: vercel.json (hunk 1 above)
Comment (JosiasAurel) [reply vercel.json:1]: i haven't experienced any unintended side effects using this. i've changed the configuration to only cache those json files still

**Pull request merged**